### PR TITLE
Home - Masthead

### LINKED ISSUE #1: Home - Masthead
Develop the home masthead for the blog according to our design sketch.

**Requirements**
- Make sure the view looks good on mobile, tablet, and mobile screens.
- Make sure the images used are optimized by using the nextjs Image component. 
- Add a heading with the copy "Fresh Beginnings".
- Add a subheading with the copy "Journey to living a toxin-free life".
- On mobile screens add a button that says about me that will eventually self-scroll the user to the about me section.

**Design Sketch**

Image

Image

### PR DESCRIPTION
**Pull request by Diabeticodes**
# Why
Styles for the alignment of the logo over the ImageMasthead component have been finalized and applied to display appropriately on all devices

## What
ImageMasthead was rebuilt to include the logo as part of it's structure instead of the logo being imported separately. This resulted in styling it easier and relative to the parent container instead of the page per se

## Testing Notes (if applicable)
There is one area on 'sm' devices right before the breakpoint where the logo begins moving left, should double check this to see a better way as I've applied what I feel to be a lot of position-styled classes to the logo div.

## Checklist

- [X] Looks good on large screens
- [X] Looks good on mobile screens

### CODE REVIEW

File: src/components/HomeMasthead/ImageMasthead.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import Image from 'next/image';
+import blogPlaceholder from '/public/images/blog-placeholder.jpg';
+import LogoImage from '/public/images/fresh-beginnings-logo.png';
+import michelleHome from '/public/images/michelle-home-min.jpg';
+import booksPlaceholderLg from '/public/images/books-placeholder-lg.jpg';
```
Comment (giantrotta24) [inline src/components/HomeMasthead/ImageMasthead.tsx]: Some nitpicky things here.
- I like separating my external imports from my internal imports, with external at the top. 
- Lets be consistent with using camel case for our image imports. 
```suggestion
import Image from 'next/image';

import blogPlaceholder from '/public/images/blog-placeholder.jpg';
import logoImage from '/public/images/fresh-beginnings-logo.png';
import michelleHome from '/public/images/michelle-home-min.jpg';
import booksPlaceholderLg from '/public/images/books-placeholder-lg.jpg';
```

File: src/pages/blog.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import type { NextPage } from 'next';
```
Comment (giantrotta24) [inline src/pages/blog.tsx:1]: I think you're good to get rid of this entire page, I started work on the look and feel for the blog in another branch.

File: src/components/About.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import Image from "next/image";
```
Comment (giantrotta24) [inline src/components/About.tsx:1]: You're good to get rid of this, I have an about me section merged to base

**Review by giantrotta24 — COMMENTED** [review]
Great work so far! Just a couple of nitpicky notes.

Also, in an effort to work through what I would tell someone at my job I wanted to point out a few things about your PR. 
- Your PR description is great! I went ahead and renamed the PR to something more descriptive to the work.
- You still had master as your base branch
Screen Shot 2022-08-28 at 8 39 28 PM
In this case we would want to be merging to `development`, but we need to make sure we're on track with base before we merge. It is good practice to check your branches are up to date with base and being merged to the right tree. You are a bit behind the `development` branch after I merged some of my work, and also need to resolve a conflict to be ready to merge. 

Regarding design, could we get rid of the circle logo and follow the design we discussed last time we spoke with the square and subheading? Something more like this
Screen Shot 2022-08-28 at 8 43 50 PM

I also noticed at certain screen sizes there is a bit too much room between the masthead and latest posts, just a big empty space of white. Could we get rid of some margin there? 
Screen Shot 2022-08-28 at 8 46 16 PM

**Comment by Diabeticodes** [comment]
Closing to reopen in a new branch from development

**Pull request closed without merging**